chart/kubeapps/templates/mongodb-secret-bootstrap-serviceaccount.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```

Comment:
we should rename the filenames to match the naming, i.e. mongodb-jobs-bootstrap-serviceaccount here